jre_emul/Classes/J2ObjC_common.h

```diff
@@ -342,5 +343,24 @@ CLASS *instance, NS_RELEASES_ARGUMENT TYPE value) J2OBJC_METHOD_ATTR { \
   - (id)autorelease { return self; }
 #endif
 
+/*
+ * iOS Application must implement j2objc_getSystemResourceBundle()
+ * for Java Systen Resource Budnle.
+ * Sample)
+ * NSBundle* j2objc_getSystemResourceBundle() {
+ *   static NSBundle *j2objc_bundle = NULL;
+ *   if (j2objc_bundle == NULL) {
+ *     NSURL* url = [[NSBundle mainBundle] URLForResource:@"j2objc_resources" withExtension:@"bundle"];
+ *     if (url != NULL) {
+ *       j2objc_bundle = [NSBundle bundleWithURL:url];
+ *     }
+ *     if (j2objc_bundle == NULL) {
+ *       j2objc_bundle = [NSBundle mainBundle];
+ *     }
+ *   }
+ *   return j2objc_bundle;
+ * }
+ */
+FOUNDATION_EXPORT NSBundle* j2objc_getSystemResourceBundle();
```

Comment:
In the past, a iOS UI developer insists to separate non-UI resource bundle with others. That is why I made that method.
I will be rollback my change. 
Someday, let's talk about how to handle such a situation.

Reply:
I'm certainly open to loading an optional bundle in some other manner, just not a required new function. Perhaps the right solution is to search all bundles, though, rather than just the main one.